array-checker.js

```diff
@@ -0,0 +1,13 @@
+function uniqueCalc(arr1, arr2) {
+  let unique1 = arr1.filter((o) => arr2.indexOf(o) === -1);
```

Comment:
Good use of _arrow functions here_ 👏🏻

As discussed before, try to always be descriptive with your identifiers names, the param `o` in `filter`'s function isn't. Maybe a better name would be `elem`.

On a side note here instead of using `indexOf` and comparison, you can use `includes`.

```js
const unique1 = arr1.filter(elem => arr2.includes(elem));
```